Include/cpython/pytime.h

```diff
@@ -0,0 +1,36 @@
+// PyTime_t C API: see Doc/c-api/time.rst for the documentation.
+
+#ifndef Py_LIMITED_API
+#ifndef Py_PYTIME_H
+#define Py_PYTIME_H
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef PY_INT64_T PyTime_t;
```

Comment:
PY_INT64_T is now always defined as `#define PY_INT64_T int64_t`, so:

```suggestion
typedef int64_t PyTime_t;
```